.github/workflows/notebooks-digest-updater-upstream.yaml

```
@@ -4,16 +4,15 @@ on:
     inputs:
       branch:
         required: true
-        description: "Provide the name of the branch you want to update ex main, vYYYYx etc"
-      release-n:
-        required: true
-        description: "Provide release N version of the notebooks ex 2023a"
-  schedule:
-    - cron: "0 0 * * 5" #Scheduled every Friday
+        description: "Provide branch name: "
```

```suggestion
        description: "Provide the name of the branch you want to update ex main, vYYYYx etc"
```

I think this is more explicit.
can we get this ?

---

done!